.github/workflows/unix_unit_tests.yml

```diff
@@ -13,7 +13,7 @@
       strategy:
         matrix:
           os: [ubuntu-latest,macos-latest]
-          python-version: [3.8, 3.9, '3.10', 3.11, 3.12]
+          python-version: [3.8, 3.11, 3.12]
```

Comment:
could you document the reason for this change on github so future developers know why it was done?

Reply:
Should we document it at the top of this PR?

Reply:
doesn't matter as long as it's somewhere on github! If a decision was made during a private meeting, it's generally good practice to document the reasoning on github